salt/metalk8s/orchestrate/deploy_new_node.sls

```diff
@@ -0,0 +1,65 @@
+{%- if pillar['bootstrap_id'] %}
+{%-   set control_plane_ips = salt.saltutil.runner('mine.get', tgt=pillar['bootstrap_id'], fun='control_plane_ip') %}
+{%- else %}
+{%-   set control_plane_ips = {} %}
+{%- endif %}
+
+{%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
+{%-   set control_plane_ip = control_plane_ips[pillar['bootstrap_id']] %}
+{%- else %}
+{%-   set control_plane_ip = 'localhost' %}
+{%- endif %}
+
+
+Deploy salt-minion on a new node:
```

Comment:
Don't you need to accept the minion key after this step?

Reply:
Hmm yes true. I will add it.

Reply:
For now ;) I hope over time we can do https://github.com/scality/metalk8s/issues/744#issuecomment-483793917 and then, instead of accepting keys here, `wait_for_event` of the minion `start` event.

Reply:
Yes indeed but for the moment I will do a classic accept